### ISSUE TITLE
Microsoft Word: NVDA Doesn't read alt text in canvas object

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4596]
**Reported by jhomme on 2014-11-04 17:48**
Even though the alt text property is filled out, all NVDA reads is a single slash for a canvas object. Tested in Word 2010 and 2013.

**Comment by bhavyashah** [comment]
@michaelDCurran stated in https://github.com/nvaccess/nvda/issues/4596#issuecomment-155329919 that he would investigate the technicals of the reported issue further. Since there was no follow-up from Mick or someone else post that initial comment, this is just a friendly reminder for @michaelDCurran and others to kindly consider resuming said technical investigation as and when time permits. Once this has been done and we have an idea of the involved implementation cost, we shall be able to assign this ticket an appropriate priority.

**Comment by Adriani90** [comment]
@ehollig could you please upload the attachment from issue tracker? Thanks.

**Comment by ehollig** [comment]
@Adriani90 the attachment is now in https://github.com/nvaccess/nvda/issues/4596#issuecomment-155329918 or right here: [canvas-accessibility.zip](https://github.com/nvaccess/nvda/files/2730745/canvas-accessibility.zip)

**Comment by Adriani90** [comment]
This is still reproucible also in Office 2016 / 365.
@michaelDCurran this issue is solved when using UIA in MS Word, the alt text is even reported twice.

**Comment by Adriani90** [comment]
Closing in favor of #6134 which contains more broader discussion.
